mat-process-api.Tests/CleanTestDb.cs

```diff
@@ -2,31 +2,35 @@
 using Microsoft.EntityFrameworkCore;
 using NUnit.Framework;
 using mat_process_api.V1.Infrastructure;
+using MongoDB.Bson;
+using MongoDB.Driver;
 
 namespace UnitTests
 {
     public class DbTest
     {
-        protected UhContext _uhContext;
+        protected IMongoDatabase mongoDatabase;
+        protected IMongoCollection<BsonDocument> collection;
 
         [OneTimeSetUp]
         public void RunBeforeAnyTests()
-        {
-            DbContextOptionsBuilder builder = new DbContextOptionsBuilder();
-
-            string TEST_UH_URL = Environment.GetEnvironmentVariable("TEST_UH_URL") ??
-                                 @"Server=localhost;Database=uhsimulator;User Id='sa';Password='Rooty-Tooty';";
-
-            builder.UseSqlServer(TEST_UH_URL);
-
-            _uhContext = new UhContext(builder.Options);
-            _uhContext.Database.BeginTransaction();
+        { 
+            string MONGO_CONN_STRING = Environment.GetEnvironmentVariable("MONGO_CONN_STRING") ??
+                                 @"mongodb://localhost:1433/?gssapiServiceName=mongodb";
```

Comment:
Same here with the connection option - do we need the "?gssapiServiceName=mongodb"?